frontend/src/components/UserTables/RidesTable.tsx

```diff
@@ -23,6 +25,62 @@ const RidesTable = ({ rides, hasButtons }: RidesTableProps) => {
   const [reassign, setReassign] = useState(false);
   const [deleteOpen, setDeleteOpen] = useState(-1);
 
+  const dayAsString = ['Sun', 'Mon', 'Tue', 'Wed', 'Thu', 'Fri', 'Sat'];
+
+  const isAvailableOnDay = (driver: DriverType, day: number) => {
+    for (const [key, value] of Object.entries(driver.availability)) {
+      if (key === dayAsString.at(day) && value != undefined) {
+        return true;
+      }
+    }
+    return false;
+  };
+
+  // You can add additional filters to filter our drivers to reassign here.
+  const reassignDriverFilters: ((driver: DriverType) => boolean)[] = [
+    // (driver: DriverType) => {
+    //   return driver.firstName != 'Bin laden';
```

Comment:
Please dont add this lol 
Specifically line 42. Change the first name example

Reply:
Yup, forgot to removed it when I was messing around. Changed it now tho.

Reply:
Don't forget to push your changes 🙏 The test driver name is still there lol